study.md

```diff
@@ -40,14 +47,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+<!-- It means to save the variable values with the method that was applied to it. For instance if I use .sort in a variable without the exclamation point it sorts the data; and when I call the variable it will show in its original form (not sorted). Using ! in sort will save the variable with its values sorted and when called the variable will have same data but sorted.  -->
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+# With @- That means that variable with @ will be available in all the methods defined in one class.
```

Comment:
That's for declaring an instance variable, not instantiating an instance of a class.  `class_instance = ClassName.new`.